h2o-core/src/main/java/water/fvec/FileVec.java

```diff
@@ -194,7 +194,7 @@ public static int calcOptimalChunkSize(long totalSize, int numCols, long maxLine
     }
     else {
       // New Heuristic
-      int minNumberRows = 10; // need at least 10 rows (lines) per chunk (core)
+      int minNumberRows = 200; // need at least 200 rows (lines) per chunk (core) - that's needed for files with headers
```

Comment:
Ok, let's keep it as it is for now, but we need to remove that 200^ limit and do the right thing.